data/types.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
A little unclear how useful these are, but like you said they could easily be converted to TS if we want.

Reply:
Yeah, basically just hints to the IDE...if your IDE supports them. makes autocomplete and other checks work.